odoo/api.py

```diff
@@ -349,7 +349,7 @@ def model_create_multi(method):
 
 def _call_kw_model(method, self, args, kwargs):
     context, args, kwargs = split_context(method, args, kwargs)
-    recs = self.with_context(context or {})
+    recs = self.with_context(context) if context else self
```

Comment:
Surprising that this was not spotted before. By the way, this should also make it:

```suggestion
    recs = self.with_context(context or **{})
```

Reply:
Well, the current logic is to give the context as `params.kwargs.context` for `call_kw `routes and as `params.context` to specify the `request context`.  I'd have liked to remove the call_kw side and give everything as params.context only, but the change would be more risky.